reactDashboard/src/components/Footer/Footer.css

```diff
@@ -66,3 +71,31 @@
 footer {
   padding-bottom: 30px;
 }
+
+.darkmode footer {
+  background-color: #121212;
+  padding-bottom: 30px;
+}
+
+.darkmode footer .sitemap .entry {
+  color: #f2f2f2;
+  font-weight: normal;
+}
+
+.darkmode footer .sitemap .title {
+  color: #c1def1;
+}
+
+.darkmode .footer-base {
+  background-color: #f2f2f2;
+  opacity: 0.75;
```

Comment:
For a fixed (ie non-hover/tooltip element), don't use opacity to make a lighter/darker colour - just adjust the background-color/color to the shades you want.

Reply:
✅